### PR TITLE
fixed typo in ContainerProperties

### PR DESCRIPTION
**Pull request by gyugyu90**
- fixed variables and methods which include typo `timout` to `timeout`

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@gyugyu90 Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-kafka&pullRequestId=1671)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-kafka&pullRequestId=1671) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@gyugyu90 Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-kafka&pullRequestId=1671)!

File: spring-kafka/src/main/java/org/springframework/kafka/listener/ContainerProperties.java [hunk 1]
```diff
@@ -559,18 +559,18 @@ public void setMicrometerTags(Map<String, String> tags) {
 		return Collections.unmodifiableMap(this.micrometerTags);
 	}
 
-	public Duration getConsumerStartTimout() {
-		return this.consumerStartTimout;
+	public Duration getConsumerStartTimeout() {
```
Comment (artembilan) [inline spring-kafka/src/main/java/org/springframework/kafka/listener/ContainerProperties.java:562]: We can't do this unconditionally: it is a breaking change.
Even if it is a bug, some projects may rely on calling this API, therefore changing names for these method will cause a compilation (and execution) problem in those target applications.

We may consider to accept the change as is for the current `2.7` version, but we definitely should consider to have a `@Deprecated` variant for the previous `2.6.x` version. Although I would stay with deprecated variants until the next minor (or major) version...

**Review by artembilan — CHANGES_REQUESTED** [review]

File: spring-kafka/src/main/java/org/springframework/kafka/listener/ContainerProperties.java (hunk 1 above)
Comment (gyugyu90) [reply spring-kafka/src/main/java/org/springframework/kafka/listener/ContainerProperties.java:562]: You're right. I restored the methods and marked them deprecated

**Review by artembilan — CHANGES_REQUESTED** [review]
Please, add your name to the `@author` list and we will merge it shortly.

**Comment by artembilan** [comment]
@gyugyu90 ,

Thank you for the contribution; looking forward for more!

**Comment by artembilan** [comment]
... and cherry-picked to `2.6.x`.

**Pull request merged**